src/components/Header/utils/encoreCatalogLogOutTimer.ts

```diff
@@ -74,9 +77,12 @@ function EncoreCatalogLogOutTimer(
         const lastVisitedValidDomainTime = Cookies.get(
           "VALID_DOMAIN_LAST_VISITED"
         );
-        const timeUntilLogOut = lastVisitedValidDomainTime
+        const timeUntilLogOut = lastVisitedValidDomainTime[
+          "VALID_DOMAIN_LAST_VISITED"
+        ]
```

Comment:
Is `lastVisitedValidDomainTime` a hash? Looks like `Cookies.get(name)` returned the cookie string value before.